lambda/translcore.ml

```diff
@@ -553,6 +593,7 @@ and transl_exp0 ~in_new_scope ~scopes e =
              ap_loc=Loc_unknown;
              ap_func=Translobj.oo_prim "copy";
              ap_args=[self];
+             ap_position=Apply_tail;
```

Comment:
Same as the reasoning for `new` except stupider since there is no particular reason to care if the call to `CamlinternalObj.copy` is tail-called. I think I'll remove this one.